src/AppBundle/Entity/User.php

```diff
@@ -13,7 +13,7 @@
  * @ORM\Table()
  * @ORM\Entity(repositoryClass="AppBundle\Entity\UserRepository")
  */
-class User implements UserInterface
+class User implements UserInterface, \JsonSerializable
```

Comment:
I guess JsonSerializable Interfase mean that all data in class can be represented as a json, not just clid.